lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -84,7 +84,7 @@ public class PuppeteerLaunchTests : PuppeteerBaseTest
                                                                             TestConstants.ChromiumRevision))
             using (var page = await browser.NewPageAsync())
             {
-                var neverResolves = page.EvaluateFunctionHandle("() => new Promise(r => {})");
+                var neverResolves = page.EvaluateFunctionHandleAsync("() => new Promise(r => {})");
```

Comment:
No reason to this change right?

Reply:
Friendly reminder @Meir017  :)

Reply:
the method `EvaluateFunctionHandle` was renamed to `EvaluateFunctionHandleAsync`

Reply:
_facepalm_ you're right